sdk/core/core-https/rollup.base.config.js

```diff
@@ -81,7 +81,7 @@ export function browserConfig(test = false) {
       cjs({
         namedExports: {
           chai: ["assert"],
-          "@opentelemetry/api": ["CanonicalCode", "SpanKind", "TraceFlags"]
+          "@opentelemetry/api": ["CanonicalCode", "SpanKind", "TraceFlags", "getSpan", "StatusCode"]
```

Comment:
we should be removing "CanonicalCode" right? since it no longer exists?

Reply:
Yes, should be able to clean this up some as well. I looked but it appears that (at least for the projects I'm modifying) they do not share a common base. So it seems like I have to propagate this bit of rollup to every rollup.base.config.js that uses core-tracing....